lib/utils/object/clear_empties.ts

```diff
@@ -0,0 +1,16 @@
+import {GeneralObjectOrValue} from "./types";
+
+export const clearEmpties = <T>(object: GeneralObjectOrValue<T>): GeneralObjectOrValue<T> => {
+	// TODO: implement deepClone for immutability of the passed in object?
```

Comment:
I don't think any of the other `utils/object` methods do this, do they?

Reply:
```suggestion
```

Reply:
I don't believe so. I'll leave this as is.